UI/import_csv/import_csv.html

```diff
@@ -44,6 +44,7 @@
                  value = request.description
                  } ?>
         </div>
+        <?lsmb IF request.type == "gl" || request.type == "inventory"  ?>
```

Comment:
Would it be an idea to put this in the controller (import_csv.pl) instead of in the view (UI/ template)?